### PR TITLE
post: add blog post about yarn

### PR DESCRIPTION
**Pull request by benmann**
Wrote something together.. feel free to add/remove.
Also.. could close https://github.com/bower/bower.github.io/pull/268
cc @sheerun @dvidsilva

### CODE REVIEW

File: _posts/2016/2016-10-12-using-bower-with-yarn.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+to the Bower client. That means you can continue using your existing `bower.json` just like before.
+Here's a quick look at some more goodies Yarn has to offer in comparison to the current Bower client.
+
+### Lockfile
+Faithful users of Bower will be very pleased to see that Yarn adds the long awaited lockfile feature with even stricter
+reproducability across devices and systems than npm offer at the moment.
+
+### Speed and offline
+Yarn's cache is a pretty elaborate improvement over what Bower had to offer so far, reducing (cold) install times by a great
+deal and even more so in a chached scenario. At the same time it allows complete installs even without an internet connection!
```
Comment (riyadhalnur) [inline _posts/2016/2016-10-12-using-bower-with-yarn.md]: typo `chached` -> `cached`

File: _posts/2016/2016-10-12-using-bower-with-yarn.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+As a bonus Yarn will retry network requests during an installation in case a single dependency couldn't be resolved right away.
+
+### Security
+Checking each packages checksum before they are executed guarantees a certain degree of integrity and gives
```
Comment (riyadhalnur) [inline _posts/2016/2016-10-12-using-bower-with-yarn.md]: `before they are installed` sounds better in this context.

File: _posts/2016/2016-10-12-using-bower-with-yarn.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+the developer more confidence from a security standpoint.
+
+### Licenses
+Being able to see every dependencies License is an often needed requirement in corporate scenarios.
```
Comment (riyadhalnur) [inline _posts/2016/2016-10-12-using-bower-with-yarn.md]: The ability to list a dependencies' license type is often required in enterprise environments. Yarn offers an easy to list the license type for a given dependency.

File: _posts/2016/2016-10-12-using-bower-with-yarn.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Yarn offers an easy way list all depedencies licenses.
+
+### Important side note
+It is important to keep in mind that just like npm, Bower consists of more than just the CLI component and
+switching between the npm and Bower ecosystem can still be improved in a number of ways.
+
+As it stands right now there still seem to be [some issues]() regarding Bower support. We are however confident that
```
Comment (riyadhalnur) [inline _posts/2016/2016-10-12-using-bower-with-yarn.md]: Link is blank. Intentional?

**Comment by benmann** [comment]
@riyadhalnur good catches! All fixed 🍬

**Comment by sheerun** [comment]
@benmann @riyadhalnur @julmot I've improved the post a little. Could you review one last time?

**Pull request merged**